src/codewhisperer/commands/basicCommands.ts

```diff
@@ -114,7 +114,7 @@ export const reconnect = Commands.declare('aws.codeWhisperer.reconnect', () => a
     await AuthUtil.instance.reauthenticate()
 })
 
-export function get(key: string, context: vscode.Memento): any {
+export function getState(key: string, context: vscode.Memento): any {
     return context.get(key)
 }
```

Comment:
What is the point of this function?

Reply:
I thought it was about naming convention but I came to know about implementation of global state. 
Instead of storing global state we need to this prompt settings. 
I will check about this **Prompt Settings.**